modal/partial_function.py

```diff
@@ -332,6 +334,7 @@ def _asgi_app(
     *,
     label: Optional[str] = None,  # Label for created endpoint. Final subdomain will be <workspace>--<label>.modal.run.
     custom_domains: Optional[Iterable[str]] = None,  # Deploy this endpoint on a custom domain.
+    requires_proxy_auth: bool = False,  # Require Proxy-Authenticate HTTP Headers on requests to the endpoint
```

Comment:
`Proxy-Authenticate` is a response header. This requires the `Proxy-Authorization` request header.

Reply:
👍